SPECS/linstor-client.spec

```diff
@@ -0,0 +1,43 @@
+Summary: DRBD distributed resource management utility
+Name:    linstor-client
+Version: 1.19.0
+Release: 1%{?dist}
+License: GPLv3
+URL:     https://linbit.com/linstor/
+
+BuildArch: noarch
+
+Source0: https://pkg.linbit.com//downloads/linstor/%{name}-%{version}.tar.gz
+
+%define python3_installed %(which python3 2>/dev/null)
+
+# The build container for XCP-ng 8.2 doesn't have python3, so python3 available
+# means we're building for XCP-ng 8.3 and use python3, python2 otherwise
+%if "%(which python3 2>/dev/null)"
+%define python "python3"
+BuildRequires: python3-setuptools
+%else
+%define python "python2"
+BuildRequires: python-setuptools
+%endif
```

Comment:
The presence of python is not guaranteed in a truly minimal build environment (such as the one where the SRPM is initially built in koji), so we can't call `which` to determine which version of the BuildRequires we want.

This is one of the reasons why we should simply have a version of the spec file for XCP-ng 8.2, which only builds with python2, and one for XCP-ng 8.3+, which only builds with python3.

Reply:
Fixed in new separated PRs for [8.2](https://github.com/xcp-ng-rpms/linstor-client/pull/2) and [8.3](https://github.com/xcp-ng-rpms/linstor-client/pull/3)